README.md

```diff
@@ -47,7 +47,7 @@ SDK package installation commands may differ depending on the environment where
 **NOTE:** The SDK also requires OpenSSL 1.0.1+ in order to support TLS1.1 & 1.2
 
 ##### 1. Typical Installation
-This is the recommended way to install the SDK. The installation is done from [PyPI](https://pypi.org/) and [Automation SDK Python Github](https://github.com/vmware/vsphere-automation-sdk-python) repositories.
+This is the recommended way to install the SDK. The installation is done from [PyPI](https://pypi.org/) and [Automation SDK Python Gitlab](https://gitlab.eng.vmware.com/vapi-sdk/vsphere-automation-sdk-python) repositories.
```

Comment:
Revert this change. It should be github not gitlab

Reply:
Done